data/duke.txt

```diff
@@ -1,2 +1 @@
-
```

Comment:
Maybe can consider putting the data file inside gitignore since it's changes are technically not part of the code!